### PR TITLE
Bugfix/work 20718 include tags missing loading finished network status2

### PR DESCRIPTION
**Pull request by phanikumar1**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,5 +39,9 @@
         "har-validator": "5.x.x",
         "http2": "3.x.x",
         "mocha": "^9.2.0"
+    },
+    "engines": {
```
Comment (phanikumar1) [inline package.json]: open source library is not restrictive. So, let's follow that and not require this in engines.

File: .npmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+v12
```
Comment (phanikumar1) [inline .npmrc]: what is this v12?

File: package.json (hunk 1 above)
Comment (joshueis) [reply package.json]: the build breaks with a version of node lower than 12, this was the motivation to restrict it. If we are against it we should add a .nvmrc with the version of node that we recommend

File: .npmrc (hunk 2 above)
Comment (joshueis) [reply .npmrc]: I believe this should be .nvmrc

**Review by joshueis — APPROVED** [review]
@ShivaniMangal  I'm approving this, it looks good.
@phanikumar1  I would still suggest keeping the engine restriction as building with node version prior to 12 fails.

**Pull request merged**